ambry-clustermap/src/test/java/com.github.ambry.clustermap/HelixClusterManagerTest.java

```diff
@@ -283,6 +283,7 @@ public void clientInitiatedLivenessChangeTest() throws Exception {
 
   /**
    * Test that the changes to the sealed states of replicas get reflected correctly in the cluster manager.
+   * This also tests multiple InstanceConfig change callbacks and that they are dealt with correctly.
```

Comment:
should we also test that multiple callbacks with the flag `INIT` are handled correctly (which is the original problem)? 

The intent is that - if in the future changes are made that revert this one, some test should fail to indicate that the change is bad.